modules/users/tests/server/user.server.model.tests.js

```diff
@@ -129,19 +129,24 @@ describe('User Model Unit Tests:', function () {
       });
     });
 
-    it('should not be able to save different user with the same email address', function () {
+    it('should not be able to save another user with the same email address', function (done) {
+      // Test may take some time to complete due to db operations
+      this.timeout(10000);
```

Comment:
They take more than 2 seconds which is mocha default timeout and they usually fail like that on travis-ci. I don't mind having 10 seconds there as a timeout (it's not a delay) just in case because we're not testing mongodb or travis's servers here.